system-preparation/README.md

```diff
@@ -4,7 +4,7 @@ Note that these input files were rapidly prepared using available tools in order
 
 ## Projects
 
-SARS-CoV and SARS-CoV-2 are very similar viruses, so we are using the available structural data from SARS-CoV (both separate simulations and homology-derived models) to aid in understanding of potential SARS-CoV-2 drug targets.
+SARS-CoV and SARS-CoV-2 are very similar viruses, so we are using the available structural data from both viruses (both separate simulations and homology-derived models) to aid in finding potential SARS-CoV-2 drug targets, as well as, understanding why SARS-CoV-2 is fundamentally different from SARS-CoV.
```

Comment:
I'm not sure "finding potential drug targets" is correct, since the proteins are the targets!

Reply:
Maybe "finding druggable sites on SARS-CoV-2 proteins".  Just wanted to make the point that we're actively looking for pockets to drug the protein targets and additionally we're trying to do some basic science to understand how CoV-1 and CoV-2 are different.

Reply:
That's perfect, thanks @Mickdub !